Ceres (Update): Restock boxes + various tweaks

arimah opened the pull request:
## About the PR
Main changes:
* Expanded the freezer by one tile so I could fit a rack with restock boxes for the ChefVend, Plasteel Chef, Booze-O-Mat and condiment station.
* Also expanded the dock/entry area for symmetry. Put down a sofa in the absence of any better ideas.
* Rebranded the ship from NC Ceres to SBB Ceres. It's now made by Shipyard BlueBird, alongside the Lyrae and the Bookworm.
* Added a pre-flight checklist as is standard on all BlueBird ships. :)

Tweaks:
* Add frames around all buttons, for better visibility. Also moved the bar shutter button for increased reachability.
* Renamed a few things to make their purpose clearer.
* Correct lateral lights, in accordance with BlueBird safety requirements. :)

## Why / Balance
Restock boxes were requested by @dvir001 as these are no longer available from cargo and it kinda sucks to run out of the items in these vending machines. Other changes were aimed at enhancing the experience.

These tweaks have no appreciable effect on the ship's appraisal price after purchase, so it remains at $49,500.

## How to test
Buy yourself a Ceres!

## Media
Preinit:
![image_2024-06-03_14-15-49](https://github.com/new-frontiers-14/frontier-station-14/assets/30327355/e7095227-7d72-472c-bb83-b0faf2b43de0)

Postinit:
![image_2024-06-03_14-19-27](https://github.com/new-frontiers-14/frontier-station-14/assets/30327355/4d6c47a5-4120-4d86-8cb7-067c30da0c48)

Freezer with restock boxes (one of each):
![image_2024-06-03_14-16-44](https://github.com/new-frontiers-14/frontier-station-14/assets/30327355/c028b5b6-cc80-44ff-8f92-56f2282d04d9)

New dock area:
![image_2024-06-03_14-16-26](https://github.com/new-frontiers-14/frontier-station-14/assets/30327355/284f055d-dadf-484c-b8fa-10bf79f66ab2)

Service techfab and materials:
![image_2024-06-03_14-17-01](https://github.com/new-frontiers-14/frontier-station-14/assets/30327355/c5538583-f949-4f27-9a51-be97f9822167)

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
none

**Changelog**
:cl:
- tweak: The Ceres is no longer under NanoTrasen branding: look for SBB Ceres instead.

dvir001 commented:
One small change that I notices that I think will help a lot is to add either floor decal or wall signs with "Food" on them to the main airlock entry since there is no way to tell its a food ship by looking at the dock itself when its parked to places.

arimah commented:
> add either floor decal or wall signs with "Food" on them
I could do that. These are so many nice colourful signs for all kinds of departments, even a zombie lab sign (?!), but nothing at all for a kitchen, restaurant or mess hall. I guess a generic directional "food" sign will have to suffice. I'd rather avoid floor decals if possible, as they look a bit too tacky for what's meant to be a higher-end restaurant.

> service techfab to replace the Plasteel Chef
Looking at what these two offer, the Plasteel Chef has one thing that the service techfab lacks: beakers. These are fairly necessary when working in a kitchen. The Plasteel Chef also has colourful mugs, but not sure if anyone ever uses them. That said, if you want a service techfab in that corner, I'll put it there.

dvir001 commented:
> beakers

beakers can just be added to the techfab, no issue, if you want you can even do it in this PR.

dvir001 commented:
Also do not use checklist, use game guidebook
Its impossible to translate it if you use a generic paper without editing the ship map.

arimah commented:
> Also do not use checklist, use game guidebook

Might wanna update the ship submission guidelines, in that case :)

dvir001 commented:
> > Also do not use checklist, use game guidebook
> 
> Might wanna update the ship submission guidelines, in that case :)

We didnt get around to it yet, but we did started already to remove notes and replace with guidebook.

MagnusCrowe commented:
![image](https://github.com/new-frontiers-14/frontier-station-14/assets/124747282/e4081c95-62ba-4d06-8ff4-cfda9ee8d8a6)
I've updated the ship submission guidelines.

arimah commented:
@dvir001 @MagnusCrowe 
I've updated the PR according to your comments now, with all new screenshots. :) The new guidebook page follows the style established by the others, with an additional section containing hints on how to use the Ceres:

![image](https://github.com/new-frontiers-14/frontier-station-14/assets/30327355/e7e061cc-6f27-42f7-801c-7a2906a43d20)
![image](https://github.com/new-frontiers-14/frontier-station-14/assets/30327355/58a4b7d0-f5c2-431d-b89a-a94f39189efd)

I've also replaced the Plasteel Chef with a service techfab and added beakers to its recipes. I'm not a huge fan of how the stacks of materials are just on the ground, but there's no usable surface sufficiently close to the techfab. I'd like people to know what the materials are for without having to get confused first.

arimah commented on Resources/ServerInfo/_NF/Guidebook/Shipyard/Ceres.xml:
```diff
@@ -0,0 +1,107 @@
+
+  "A medium-size, high-class restaurant ship with ample seating, integrated botany and a dining room for VIP guests."
+
+  [italic]Brought to you by BlueBird Starship Design & Construction.[/italic]
```
Note: This lone `&` is invalid XML, but the parser accepts it. If I write the correct `&amp;` instead, then the guidebook shows the literal text `Design &amp; Construction`. C'est la vie.

dvir001 commented:
My only issue now is that im forced to update the McCargo to fight back (´。＿。｀)
